### DISCUSSION TITLE
Breeze theme Iphone 12 and lesser version Click event not working

### DISCUSSION THREAD

**Discussion by web01bharat** [breezefront/community#292] [1. Help]
Hi support,

We have recently faced an issue with the iPhone 12, while we click on any events the action does not perform. Like: the Close button does not click, the Menu menu does not open.
Magento 2 Adobe Commerce ver. 2.4.6-p6
Swissup_Breeze 2.17.8

![Screenshot Capture - 2024-06-27 - 17-55-09](https://github.com/breezefront/community/assets/135975409/ad0799d1-abb7-423f-a8af-547b80b7be92)

P

Thanks 
Bharat

**Comment by vovayatsyuk** [comment]
Please provide a URL to your site. Otherwise, I can't help you.

**Comment by web01bharat** [comment]


**Reply by vovayatsyuk** [reply]
For me, it's working fine. Tested using iPhone 12 mini.

**Comment by web01bharat** [comment]
please check the video attached 
we have check in breeze blank as well 
[click.webm](https://github.com/breezefront/community/assets/135975409/4e818375-23e7-4295-bd85-751c7cf49bc0)

**Reply by vovayatsyuk** [reply]
It looks like you are testing against Safari 14. Mine is 17. Sorry, but I don't have a device to see what is wrong here.
